cpt/config.py

```diff
@@ -5,6 +5,9 @@ def __init__(self, conan_api, printer):
         self._conan_api = conan_api
         self.printer = printer
 
-    def install(self, url):
-        self.printer.print_message("Installing config from address %s" % url)
-        self._conan_api.config_install(url, verify_ssl=True)
+    def install(self, url, args=None):
+        message = "Installing config from address %s" % url
+        if not args is None:
```

Comment:
`if args:` is enough in this case. FYI the pythonic comparison with None is `if args is not None`   
```suggestion
        if args:
```

Reply:
done,
thx for the tip